src/IssueViz.Security.UnitTests/OpenInIDE/Api/OpenInIDERequestHandlerTests.cs

```diff
@@ -266,29 +267,6 @@ await testSubject.ShowHotspotAsync(ValidRequest)
             logger.AssertPartialOutputStringExists(hotspotFilePath, hotspotStartLine.ToString());
         }
 
-        [TestMethod]
-        public async Task ShowHotspot_DataIsValid_IssueFirstAddedToStoreAndThenSelected()
```

Comment:
Test is no longer relevant since it is now possible to instead test the dependency between mocks, as done in `ShowHotspot_DataIsValid_NavigationSucceeded_And_IssueAddedToStoreAndSelected`